staging/src/k8s.io/client-go/pkg/version/base.go

```diff
@@ -51,9 +51,13 @@ var (
 	// semantic version is a git hash, but the version itself is no
 	// longer the direct output of "git describe", but a slight
 	// translation to be semver compliant.
+
+  // NOTE: The $Format strings are replaced during 'git archive' thanks to the
+  // companion .gitattributes file containing 'export-subst' in this same
+  // directory.  See also https://git-scm.com/docs/gitattributes
```

Comment:
cool, gofmt (space vs tab) and I'll approve.

Reply:
done